packages/e2e-testing/__tests__/bad-network.test.ts

```diff
@@ -118,14 +148,14 @@ describe('Payment & Receipt Managers E2E', () => {
     let numPayments = 0;
 
     while (numPayments < 20) {
-      // try {
-      await syncChannels(PAYER_SERVER_URL);
-      const {status} = await successfulPayment(PAYER_SERVER_URL);
-      if (status === 200) numPayments++;
-      // } catch (err) {
-      //   logger.trace('Payment failed', {err});
-      //   continue;
-      // }
+      try {
+        await syncChannels(PAYER_SERVER_URL);
+        const {status} = await successfulPayment(PAYER_SERVER_URL);
+        if (status === 200) numPayments++;
+      } catch (err) {
+        if (err.toString() === 'Request failed with status code 500') continue;
+        throw err;
```

Comment:
What's the reasoning for this change?

Reply:
I had mistakenly thought it might be possible for a payment to fail (no free channels) but upon thinking about it more, this should not happen with this test.